settings/settings.json

```
@@ -40,9 +40,16 @@
                         "settingName": "itemgrid.reset",
                         "type": "bool",
                         "default": "true"
+                    },
+                    {
+                        "title": "Items to Load",
+                        "description": "Default Items that are loaded is 60. If you have network issues try reducing this number to help load items faster",
+                        "settingName": "itemgrid.Limit",
+                        "type": "integer",
+                        "default": 60
```

```suggestion
                        "default": "60"
```

Values stored in the registry need to be string values